cmd/pullrequest-init/disk.go

```diff
@@ -80,28 +86,54 @@ func ToDisk(pr *PullRequest, path string) error {
 }
 
 func commentsToDisk(path string, comments []*Comment) error {
+	// Create a manifest to keep track of the comments that existed when the
+	// resource was initialized. This is used to verify that a comment that
+	// doesn't exist on disk was actually deleted by the user and not newly
+	// created during upload.
+	f, err := os.Create(filepath.Join(path, manifestPath))
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	manifest := bufio.NewWriter(f)
+
 	for _, c := range comments {
-		commentPath := filepath.Join(path, strconv.FormatInt(c.ID, 10)+".json")
+		id := strconv.FormatInt(c.ID, 10)
```

Comment:
Is this manifest intended to be human readable? Would json serializing or gob serializing make it easier to safely round trip, so we don't have to manually encode to text?

Reply:
Great idea!

Manifests are not meant to be human readable. Switched to gob encoding.